CHANGELOG.md

```diff
@@ -1,9 +1,46 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## v0.9.1
+
```

Comment:
Some added features are missing:
- trains
- sources (not sure if maybe already before)
- extended door status (temp_close, etc #533)
- install script for deps
- packaging for windows and MacOS
- improvement for escalators (#865)

Internal changes:
- reimplementation of grid based neighborhood search
- your changes to the Pedestrians, e.g., removing the subroom/room 
- event handling (iirc)

Can always be mentioned: stability improvement and bug fixes